deploy/frontend.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: acs
+objects:
+  - apiVersion: cloud.redhat.com/v1alpha1
+    kind: Frontend
+    metadata:
+      name: acs
```

Comment:
The `name` metadata here must match the route our assets are served under in the consoledot UI.

Inspecting the network tab shows that the route is `/apps/acs`, hence the `acs` values here.